packages/core/state/interaction/logics.ts

```diff
@@ -582,12 +582,22 @@ const setIsSmallScreen = createLogic({
  * Logs details of files that are being moved.
  */
 const moveFilesLogic = createLogic({
-    type: MOVE_FILES,
-    process(deps, dispatch, done) {
-        const action = deps.action as MoveFilesAction;
-        console.log(`Moving files:`, action.payload.fileDetails);
-        done();
+    async process({ action, getState }: ReduxLogicDeps, _dispatch, done) {
+        try {
+            const httpFileService = interactionSelectors.getHttpFileService(getState());
+            const username = interactionSelectors.getUserName(getState());
```

Comment:
Does this come back as potentially undefined? What is the value on web? I know the UI prevents using this on web, but adding a check for the username here if it is potentially undefined would be good type guarding

Reply:
it does potentially come back as undefined. In this situation during submission the user is set to 'anonymous'. This should not be reachable for web since it is disabled but in the strange condition that a desktop user is undefined than it will use the `anonymous` user id.